lib/stories-common.js

```diff
@@ -9,8 +9,8 @@ export const actionDefault = msg => ev => {
 
 export const Sample = ({ children, ...props }) => (
   <ExtractSource {...props}>
-    <div className="sample" onClick={actionDefault("clicked")}>
+    <React.Fragment onClick={actionDefault("clicked")}>
```

Comment:
Hmm, I don't think this onClick handler works on a React.Fragment. The handler is here to intercept clicks on links and buttons that would jump us out of the Storybook, and it looks like that stops working in the Storybook build on this PR

Reply:
@lmorchard switched to div and updated `preview-styles.css`